selfdrive/manager/manager.py

```diff
@@ -30,6 +30,8 @@ def manager_init() -> None:
   # update system time from panda
   set_time(cloudlog)
 
+  os.environ['OPENPILOIT_PREFIX'] = "MANAGER"
```

Comment:
manager should use a default prefix. here's one use case this breaks: openpilot's running and you want to run a debug script like dump.py